### PR TITLE
Rachel Cundiff Homework1

### PR DESCRIPTION
**Pull request by rcundiff**
You can see my commits to follow my process of how I built this document.  My boyfriend (a computer scientist) showed me how to use a Markdown editor which I liked a lot more than simply writing it in RStudio.  I had a lot of trouble including the image.  I finally realized that I had written .jpg, not .JPG, and apparently the capitalization matters!  Otherwise, the daringfireball.net website was extremely helpful to create a proper Markdown document.

### CODE REVIEW

**Comment by sanjanasarathy** [comment]
Evaluation HW-01 
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |    x       |              |            |
| **Coding strategy**         |      x     |              |            |
| **Presentation**            |     x      |              |            |
| **Achievement, creativity** |      x     |              |            |
| **Ease of access**          |     x      |              |            |

Remarks:

* Overall a great assignment! I loved the picture of your cat and the list of your favorite TV shows. I didn't realize you could divide up the page with the use of "--------------------------"
* You could have included # Subheadings to break up your biography, but this was a short assignment so it wasn't completely needed

**Review by sophiahk — COMMENTED** [review]
Hi Rachel!
First of all, your cat is very cute. I can see nothing wrong with your code: I would say all categories in our rubric fit either excellent or satisfactory, considering the assignment was fairly simple and you completed all of the required steps. I also had difficulty inserting an image saved from my computer - so now I know that it has to be in caps! Plus, it was useful to learn that the multiple dashes can separate sections, and you don't have to create different sections by nesting layers of headers, which is what I originally tried to do. Good luck on the next homework!

**Comment by ayseguluysal** [comment]
Evaluation HW-1 for Rachel Cundiff
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |    X       |              |            |
| **Coding strategy**         |   X        |              |            |
| **Presentation**            |    X       |              |            |
| **Achievement, creativity** |     X      |              |            |
| **Ease of access**          |    X       |              |            |

Really great assignment! Enjoyed reading about your research, and your favorite shows. You incorporated many Markdown elements nicely!

File: README.md [hunk 1]
```diff
@@ -1,2 +1,22 @@
...
+
++ Peaky Blinders
++ Man in the High Castle
++ The Dick van Dyke Show
++ The Crown
++ Law & Order: SVU
+
+I went to Loyola University Chicago for undergrad, and I'm really excited that our men's basketball team is in the final four for the NCAA tournament. **GO RAMBLERS!!**
+
+Finally, if you're interested, here is [my professional website!](https://rcundiff.github.io/)
```
Comment (ayseguluysal) [inline README.md:22]: I didn't know you could add a website link! Very creative idea, and good to know. Your readme was very easy to follow, and your coding looks great.

**Review by ayseguluysal — APPROVED** [review]

**Review by nnickels — COMMENTED** [review]
Evaluation HW-01 for Rachel Cundiff
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     X      |              |            |
| **Coding strategy**         |     X      |              |            |
| **Presentation**            |     X     |              |            |
| **Achievement, creativity** |           |      X       |            |
| **Ease of access**          |     X      |              |            |

Remarks:

* Great job on the first homework! You've successfully incorporated many of the markdown elements and created a clear, nice looking README.
* I'm not sure why you were having issues with .jpg vs. .JPG - .jpg should work so long as your markdown syntax matches the file saved in your local repo, but it may have something to do with Linux being case sensitive. Either way, great job getting it to work.

**Pull request closed without merging**